zerqu/handlers/feeds.py

```diff
@@ -91,6 +93,22 @@ def yield_entry(topic, user):
     else:
         yield u'<name>Anonymous</name>'
     yield u'</author>'
+    webpage = WebPage.cache.get(topic.webpage) or u''
```

Comment:
@lepture To make sure `webpage` would be an basestring type when running `yield u'<content type="html"><![CDATA[%s]]></content>' % (webpage + topic.html)` in line 112.